doc/lsst.ap.pipe/pipeline-with-fakes.rst

```diff
@@ -0,0 +1,194 @@
+* **Configuration overrides**: Sets injection task defaults for PSF, photometry, and WCS handling
+* **APDB configuration**: Uses placeholder config that must be user-specified at runtime
+
+Why Generate at Build Time?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This approach provides several advantages:
+
+* **Consistency**: Ensures the fakes pipeline stays synchronized with the base pipeline
+* **Maintenance**: Automatically incorporates changes to the base pipeline without manual updates
+* **Flexibility**: Allows systematic application of injection patterns across different pipeline subsets
+* **Testing**: Ensures the fakes pipeline is always available for validation and testing
+
+The generated ``ApPipeWithFakes.yaml`` file should **not** be manually edited, as it will be
+overwritten during the next package build. Instead, modifications should be made to:
+
+* The base ``ApPipe.yaml`` pipeline
+* The ``ApPipePostInjectedTasks.yaml`` injection-specific tasks
+* The ``SConstruct`` build configuration
+* Camera-specific configuration files that import the ingredients
+
+Running the Pipeline
+====================
+
+Prerequisites
+-------------
+Running `ApPipeWithFakes` requires the same prerequisites as the standard `ApPipe` pipeline, and in addition an **injection catalog**: pre-made injection catalogs should be ingested into the input collection to use.
+
+Output Products
+===============
+
+The ApPipeWithFakes pipeline produces all standard AP pipeline outputs plus additional fake-specific datasets:
+
+Standard AP Products (with ``fakes_`` prefix)
+----------------------------------------------
+
+* ``fakes_preliminary_visit_image``: Calibrated science image with injected sources
+* ``fakes_difference_image``: Difference image between injected science image and template
+* ``fakes_dia_source_detector``: Detected sources in difference image
+* ``fakes_dia_object_apdb``: Associated DiaObjects stored in APDB
+
+Fake-Specific Analysis Products
+-------------------------------
+
+* ``fakes_preliminary_visit_image_catalog``: Catalog of injected source properties
+* ``fakes_goodSeeingDiff_matchDiaSrc``: Matches between injected and detected sources
+* ``fakes_goodSeeingDiff_matchAssocDiaSrc``: Matches for successfully associated sources
+* Performance analysis results with recovery fractions by magnitude and SNR
+
+These outputs enable comprehensive validation of the AP pipeline's source detection, measurement,
+and association performance using controlled synthetic sources with known properties.
+
+Related Documentation
+=====================
+
+* :doc:`pipeline-overview`: Overview of the standard AP pipeline
+* :doc:`pipeline-tutorial`: Tutorial for running AP pipelines
+* :doc:`apdb`: Alert Production Database setup and usage
+* `source_injection package documentation <https://pipelines.lsst.io/modules/lsst.source.injection/index.html>`_: Details on source injection tasks
```

Comment:
This looks like just a dump of the `ap_pipe` TOC. Are these all actually related, and can they be moved elsewhere? For example, the tutorial would make sense as an inline link under "Running the Pipeline".